screen/budgets/src/main/java/com/ivy/budgets/BudgetViewModel.kt

```diff
@@ -176,6 +183,10 @@ class BudgetViewModel @Inject constructor(
                     )
                 }.toImmutableList()
             }
+            this@BudgetViewModel.totalRemainingBudget.doubleValue = calculateTotalRemainingBudget(
```

Comment:
no need for `this@`, if there's a clash of names rename to `totalRemainingBudgetState`